new_experiment/nbconverted/debug.py

```diff
@@ -0,0 +1,405 @@
+
+
+# In[40]:
+
+
+template_nonrecount2_nofilter_DE_stats_filename = os.path.join(
+    DE_stats_dir,
+    "DE_stats_template_data_cis-par-KU1919_real_without_filtering.txt"
+)
+
+template_nonrecount2_nofilter_DE_stats = pd.read_csv(
+    template_nonrecount2_nofilter_DE_stats_filename,
+    sep="\t",
+    index_col=0,
+    header=0
+)
+
+template_nonrecount2_nofilter_DE_stats.loc["RBPJL"]
+
+
+# In[41]:
+
+
+template_nonrecount2_nofilter_DE_stats.loc["SSTR5.AS1"]
+
+
+# **Takeaways:**
+# * Case 1: genes with only simulated statistics is because template experiment has all 0 counts
+# * Case 2: genes with fewer than 25 simulated experiments, some simulated experiments have all 0 counts for those genes
+# * Case 3 (only found using non-recount2 template): genes with missing p-value in template experiment have NaN output from DESeq2, which indicates those genes that have been automatically filtered by DESeq2 as very likely to not have no significance: https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#pvaluesNA
+# 
+# 
+# **Proposed solution:**
+# * Case 1 and 2: Remove genes with 0 counts across all samples. I will also add an option to additional remove genes that have mean counts < user specified threshold (for Jim's case). This should get rid of those rows with missing values in template statistic columns. These genes will be missing in the validation against Crow et. al. so I'll need to ignore these genes in the rank comparison. Any gene removed from the simulated experiments will get a lower `number of simulated experiments` reported so I will need to document this for the user.
+# * Case 3: (option 1) I can change the parameter setting to turn off this autofiltering and it will perform all tests and report them. (option 2) Replace padj values = "NA" with "Filtered by DESeq2" and document that this means that DESeq2 pre-filtered these genes as likely not being signicant to help increase detection power. For now I am using option 1. DESeq2 documentation states: 
```

Comment:
Will Option 1 take a lot longer to run?

Reply:
After rerunning this it doesn't seem like its adding a significant amount of time. Though I haven't performed any rigorous testing given the time constraint to complete this analysis